library/std/src/gc.rs

```diff
@@ -399,6 +392,37 @@ impl<T> Gc<T> {
             )
         }
     }
+
+    #[unstable(feature = "gc", issue = "none")]
+    pub fn unregister_finalizer(&mut self) {
+        let ptr = self.ptr.as_ptr() as *mut GcBox<T> as *mut u8;
+        unsafe {
+            bdwgc::GC_register_finalizer(
+                ptr,
+                None,
+                ::core::ptr::null_mut(),
+                ::core::ptr::null_mut(),
+                ::core::ptr::null_mut(),
+            );
+        }
+    }
+}
+
+#[cfg(profile_gc)]
+#[derive(Debug)]
+pub struct FinalizerInfo {
```

Comment:
I guess this is more `FinalizersInfo` or `GCProfileInfo` or something like that?

Reply:
Yes good point. But this existed before, this commit has just moved it around. Shall I rename it?

Reply:
I think renaming it would be a good idea, but you could do it in a subsequent PR. Up to you.